jscomp/others/js_date.ml

```diff
@@ -0,0 +1,111 @@
+external setFullYear : t -> float -> float = "" [@@bs.send]
+external setFullYearM : t -> year:float -> month:float -> float = "setFullYear" [@@bs.send]
+external setFullYearMD : t -> year:float -> month:float -> date:float -> float = "setFullYear" [@@bs.send]
+external setHours : t -> float -> float = "" [@@bs.send]
+external setHoursM : t -> hours:float -> minutes:float -> float = "setHours" [@@bs.send]
+external setHoursMS : t -> hours:float -> minutes:float -> seconds:float -> float = "setHours" [@@bs.send]
+external setHoursMSMs : t -> hours:float -> minutes:float -> seconds:float -> milliseconds:float -> float = "setHours" [@@bs.send]
+external setMilliseconds : t -> float -> float = "" [@@bs.send]
+external setMinutes : t -> float -> float = "" [@@bs.send]
+external setMinutesS : t -> minutes:float -> seconds:float -> float = "setMinutes" [@@bs.send]
+external setMinutesSMs : t -> minutes:float -> seconds:float -> milliseconds:float -> float = "setMinutes" [@@bs.send]
+external setMonth : t -> float -> float = "" [@@bs.send]
+external setMonthD : t -> month:float -> date:float -> float = "setMonth" [@@bs.send]
+external setSeconds : t -> float -> float = "" [@@bs.send]
+external setSecondsMs : t -> seconds:float -> milliseconds:float -> float = "setSeconds" [@@bs.send]
+external setTime : t -> float -> float = "" [@@bs.send]
+external setUTCDate : t -> float -> float = "" [@@bs.send]
+external setUTCFullYear : t -> float -> float = "" [@@bs.send]
+external setUTCFullYearM : t -> year:float -> month:float -> float = "setUTCFullYear" [@@bs.send]
+external setUTCFullYearMD : t -> year:float -> month:float -> date:float -> float = "setUTCFullYear" [@@bs.send]
+external setUTCHours : t -> float -> float = "" [@@bs.send]
+external setUTCHoursM : t -> hours:float -> minutes:float -> float = "setUTCHours" [@@bs.send]
+external setUTCHoursMS : t -> hours:float -> minutes:float -> seconds:float -> float = "setUTCHours" [@@bs.send]
+external setUTCHoursMSMs : t -> hours:float -> minutes:float -> seconds:float -> milliseconds:float -> float = "setUTCHours" [@@bs.send]
+external setUTCMilliseconds : t -> float -> float = "" [@@bs.send]
+external setUTCMinutes : t -> float -> float = "" [@@bs.send]
+external setUTCMinutesS : t -> minutes:float -> seconds:float -> float = "setUTCMinutes" [@@bs.send]
+external setUTCMinutesSMs : t -> minutes:float -> seconds:float -> milliseconds:float -> float = "setUTCMinutes" [@@bs.send]
+external setUTCMonth : t -> float -> float = "" [@@bs.send]
+external setUTCMonthD : t -> month:float -> date:float -> float = "setUTCMonth" [@@bs.send]
+external setUTCSeconds : t -> float -> float = "" [@@bs.send]
+external setUTCSecondsMs : t -> seconds:float -> milliseconds:float -> float = "setUTCSeconds" [@@bs.send]
+external setUTCTime : t -> float -> float = "" [@@bs.send]
+external setYear : t -> float -> float = "" [@@bs.send]
+(** DEPRECATED, use `setFullYear` instead *)
```

Comment:
I figured this was a different kind of deprecated. But quite frankly it could just as well be removed.